server/models/Profile.js

```diff
@@ -25,19 +25,30 @@ const profileSchema = new mongoose.Schema({
   },
   dateOfBirth: {
     type: Date,
-    default: null
+    default: null,
   },
   availability: {
     type: [timeSlot],
-    default: []
+    default: [],
```

Comment:
This is not required for this attribute. Note that `availability` is an array of values so pushing in another array as a default value won't make any sense.

Reply:
Changed made; no need to initialize an array here.